WTWSMS/history/titles/c_somerset.txt

```diff
@@ -34,6 +34,9 @@
 480.1.1={
 	holder = 2 # Geraint Llyngesic ap Erbin
 }
+514.1.1={
+	holder = 228000 # Cado ap Gerren
+}
```

Comment:
(I don't know much about Briton history)
Could you explain this change?

Reply:
> 
> 
> (I don't know much about Briton history)
> Could you explain this change?

Previous holder died in 514, so between his death and anglo-saxon occupation i gave these counties to heir.